docs/deploy-worker-udf-binaries.md

```diff
@@ -0,0 +1,101 @@
+# Deploy Worker and UDF Binaries General Instruction
+
+This documentation will describe how to deploy Worker and UDF (User Defined Function) binaries, which Environment Variable to set up and some commonly used options when launching applications with `spark-submit`.
```

Comment:
```suggestion
This how-to describes how to deploy Worker and UDF (User-Defined Function) binaries, including which Environment Variables to set up and some commonly used parameters when launching applications with `spark-submit`.
```

Reply:
Thanks for your suggestion. But `how-to describes how to...` seems like a bit unclear for me.

Reply:
It seems that our end-goal with these docs is to incorporate them all to the main dotnet/docs repo. I agree the wording here currently is good too! My suggestion was just to mimic the style found in official [how-to guides](https://docs.microsoft.com/en-us/dotnet/spark/how-to-guides/debug). 

@rapoth Do we want the docs on here to have the same wording as those that'll be on the official docs page?

Reply:
Cool, you are right. It's better to follow the official guides. My understanding is that all these docs will move to dotnet/docs eventually. How about the following?
```suggestion
This how-to provides general instructions on how to deploy Worker and UDF (User-Defined Function) binaries, including which Environment Variables to set up and some commonly used parameters when launching applications with `spark-submit`.
```

Reply:
I think that sounds great!